File: dotnet/Allmark/Render/StrongRenderer.cs
namespace Allmark.Render;

using Allmark.Types;

public static class StrongRenderer
{
    public static OutputRenderer Create()
    {
        return new OutputRenderer
        {
            Name = "strong",
            Render = Render,
        };
    }

    public static void Render(MarkdownNode node, RendererState state, bool? decode = true)
    {
        RenderTag.Execute(node, state, "strong");
    }
}

File: zig/src/render/strongRenderer.zig
const std = @import("std");
const MarkdownNode = @import("../types/MarkdownNode.zig").MarkdownNode;
const RendererState = @import("../types/RendererState.zig").RendererState;
const Renderer = @import("../types/Renderer.zig").Renderer;
const renderTag = @import("renderTag.zig").renderTag;

pub fn render(node: *const MarkdownNode, state: *RendererState, first: ?bool, last: ?bool, decode: ?bool) void {
    _ = first;
    _ = last;
    _ = decode;
    renderTag.renderTag(node, state, "strong", true);
}

pub const strongRenderer = Renderer{
    .name = "strong",
    .render = render,
};
